### DISCUSSION TITLE
How to disable animations for gtk-layer-shell -based app?

### DISCUSSION THREAD

**Discussion by iliabylich** [hyprwm/Hyprland#7355] [Q&A] [answered]
I have an app that uses gtk4-layer-shell, when it's active I don't see it in `hyprctl clients`:

```sh
$ hyprctl clients | grep -E 'class|title'
	class: code-url-handler
	title: hyprland.conf - dotfiles - Visual Studio Code

	class: google-chrome
	title: New Discussion · hyprwm/Hyprland - Google Chrome

	class: gnome-terminal-server
	title: ilya@debian: ~/experiments/dotfiles
```

Not sure if it's relevant but I can see it in `hyprctl layers`:

```sh
$ hyprctl layers
Monitor DP-1:
	Layer level 0 (background):
		Layer 557814341910: xywh: 0 0 1920 1080, namespace: wallpaper
	Layer level 1 (bottom):
	Layer level 2 (top):
		Layer 5578143638a0: xywh: 0 0 1920 56, namespace: bar-0
	Layer level 3 (overlay):
		Layer 55781448c070: xywh: 1420 56 500 205, namespace: gtk4-layer-shell
```

When I open it (obviously) I get a standard fade-in animation that I'd like to disable for this (and probably all other) gtk-layer-shell -based widgets. Is it possible to get it using `windowrule{,v2}`?

**Comment by hannahfluch** [comment]
Why did you mark this issue as closed? I am facing the same issue, have you found a solution?

**Comment by hannahfluch** [comment, marked as answer]
I did some reasearch and for anyone facing the same issue: you can use layerrules in the following way:
`layerrule = noanim, gtk4-layer-shell
`

**Reply by iliabylich** [reply]
Yes, it's `layerrule = noanim, <layername>` where `<layername>` is whatever is the name of the GTK4 Window widget. You can always do `hyprctl layers` to see what's currently set.
